beacon-chain/forkchoice/doubly-linked-tree/store.go

```diff
@@ -148,11 +150,23 @@ func (s *Store) insert(ctx context.Context,
 			return err
 		}
 	}
-
 	// Update metrics.
 	processedBlockCount.Inc()
 	nodeCount.Set(float64(len(s.nodeByRoot)))
 
+	// Apply proposer boost
```

Comment:
Nitpick: what if we move this above `updateBestDescendant `? I think I like this as closer to where we insert node `s.nodeByRoot[root] = n`